code/modules/jobs/job_types/warden.dm

```diff
@@ -26,6 +26,15 @@
 
 	family_heirlooms = list(/obj/item/book/manual/wiki/security_space_law)
 
+	mail_goodies = list(
+		/obj/item/storage/fancy/cigarettes = 15,
+		/obj/item/storage/box/handcuffs = 10,
+		/obj/item/storage/box/teargas = 10,
+		/obj/item/storage/box/flashbangs = 10,
+		/obj/item/storage/box/rubbershot = 10,
+		/obj/item/storage/box/lethalshot = 5
```

Comment:
hey hey, now we're talking

you might wanna cut the tear gas box from the list, though- no one ever uses those